backend/src/data/models/groups-to-permissions/groups-to-permissions.model.ts

```diff
@@ -0,0 +1,14 @@
+import { DbTableName } from '~/common/enums/enums';
+import { Abstract } from '../abstract/abstract.model';
+
+class GroupsToPermissions extends Abstract {
+  'permission_id': number;
```

Comment:
```suggestion
  'permission_id': number;
  
```
add spaces between keys (fix for all). we will try to handle it by eslint in the future

Reply:
And use camelCase as in user model